share/spice/metar/metar.js

```diff
@@ -0,0 +1,29 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_metar = function(api_result) {
+        
+        if (!api_result || api_result.Error) {
+            return Spice.failed('metar');
+        }    
+
+        Spice.add({
+            id: "metar",
+            name: 'Weather',
+            data: api_result,
+            meta: {
+                sourceName: "Aviation Weather REST API",
+                sourceUrl: "http://avwx.rest"
+            },
+            normalize: function(item) {
+                return {
+                    title: item["Station"], 
+                    subtitle: item["Time"],
```

Comment:
Sure! 

I had those improvements in mind, but I thought that for a first project it'd be nice to start with returning the bare-bones data; assuming that if a user was looking for a metar, he or she would know the technical details: the ICAO code of the airport and how to interpret the raw metar.

I'll look into showing that extra information in a nice way. I guess that the `List` template would be the best choice.

One question, are strings internationalized, or I can just display the text in English?